### ISSUE TITLE
ARM systems support

### ISSUE THREAD

**Issue by Kinderix** [josephdadams/TallyArbiter#515]
### What feature would you like to see added to TallyArbiter?

I want this feature!

Is there anyway that in the future you will provide support for Windows 10 ARM?

**Comment by hrueger** [comment]
Are you talking about the Desktop Version?
The npm package should also work on arm.

**Comment by D4rkSoul67** [comment]
I'm running Windows 10 on a Raspberry Pi 4 and the normal exe doesn't work until I realised it was x64.

**Comment by hrueger** [comment]
Definitely interesting. I've never tried that.
We can probably build the Desktop App for arm. However, unfortunately, I currently don't have time to look into this.
If you want to contribute, feel free to do so 👍

**Comment by josephdadams** [comment]
Yeah I'm sure Github actions can build whatever windows arm version is needed and include it in the tagged releases. You could also just install nodejs and run the project from source

**Comment by josephdadams** [comment]
ARM is supported now. The container images are built and published as multi-arch manifests covering amd64, arm64, armv7 and armv6, so the correct image is selected automatically on ARM hosts such as a Raspberry Pi. Closing as complete.
